.gitlab-ci.yml

```diff
@@ -0,0 +1,332 @@
+                cpu: 0.25
+                memory: 0.5Gi
+              volumeMounts:
+              - mountPath: /drupal/web/sites/default/files
+                volumeName: filesharevol
+              env:
+              - name: X_ROBOTS_TAG
+                value: noindex
+              - name: LIMITED_BETA_MODE
+                value: 1
+            - image: acreccukspre.azurecr.io/microsites-drupal-fpm:${CI_COMMIT_REF_SLUG}
+              name: drupal
+              resources:
+                cpu: 0.75
+                memory: 1.5Gi
+              volumeMounts:
+              - mountPath: /drupal/web/sites/default/files
+                volumeName: filesharevol
+              env:
+              - name: MYSQL_HOST
+                value: mariadb-ecc-uks-pre.mariadb.database.azure.com
+              - name: MYSQL_USER
+                value: mariadb-root
+              - name: MYSQL_DATABASE
+                value: drupal_microsites
+              - name: MYSQL_PASSWORD
+                secretRef: mysql-password
+              - name: OPENID_CONNECT_PARAMS
+                secretRef: openid-connect-params
+      EOF
+    - echo "Creating new revision of Container App"
+    - az containerapp revision copy --name microsites --resource-group rg-ecc-microsites-uks-pre --yaml revision.yml --subscription "Essex County Council (Microsites)"
+  rules:
+    - if: '$CI_COMMIT_BRANCH =~ /^release\//'
+      when: manual
+
+drush-deploy-preprod:
+  image: mcr.microsoft.com/azure-cli
+  stage: preproduction-postdeploy
+  needs: ["deploy-to-preprod"]
+  before_script:
+    - apk add socat
+    - az extension add --name containerapp
+    - az login --service-principal -u${ARM_CLIENT_ID} -p${ARM_CLIENT_SECRET} -t${ARM_TENANT_ID}
+    - az account set --subscription "Essex County Council (Microsites)"
+  script:
+    - timeout 1200 socat EXEC:'az containerapp exec --command /drupal/deploy.sh --name microsites --container drupal --resource-group rg-ecc-microsites-uks-pre',pty,setsid,ctty STDIO,ignoreeof
+  rules:
+    - if: '$CI_COMMIT_BRANCH =~ /^release\//'
+      when: manual
+
+deploy-to-prod:
+  image: mcr.microsoft.com/azure-cli
+  stage: production
+  needs: ["deploy-to-preprod"]
+  environment:
+    name: production
+    url: https://essex.gov.uk/
+  before_script:
+    - az extension add --name containerapp
+    - az login --service-principal -u${ARM_CLIENT_ID} -p${ARM_CLIENT_SECRET} -t${ARM_TENANT_ID}
+  script:
+    - echo "Importing image from Pre-Production into Production"
+    - az acr import --force --name acreccuksprod --source acreccukspre.azurecr.io/microsites-nginx-drupal:${CI_COMMIT_REF_SLUG} --image microsites-nginx-drupal:${CI_COMMIT_REF_SLUG} --subscription "Essex County Council (Common)"
+    - az acr import --force --name acreccuksprod --source acreccukspre.azurecr.io/microsites-drupal-fpm:${CI_COMMIT_REF_SLUG} --image microsites-drupal-fpm:${CI_COMMIT_REF_SLUG} --subscription "Essex County Council (Common)"
```

Comment:
@MatthewWilkes should the subscription be `--subscription "Essex County Council (Microsites)"` here? I was copied and pasted from .gov's gitlab-ci file.

Reply:
As above, this is right.